<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1285</width>
    <height>785</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>B Nazanin</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ادغام و سازمان‌دهی مجدد متون مشابه فارسی</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1281</width>
      <height>711</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <attribute name="label">
      <string>ورودی</string>
     </attribute>
     <widget class="QListWidget" name="lstInputFiles">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>10</y>
        <width>311</width>
        <height>461</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier</family>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'B Nazanin'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;لیست فایل‌های ورودی&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnInputAddFile">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>480</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>افزودن ...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnInputDeleteFile">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>955</x>
        <y>480</y>
        <width>161</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>حذف</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="txtBrsInputReader">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>10</y>
        <width>911</width>
        <height>491</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'B Nazanin'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;لطفا یک فایل را از لیست انتخاب کنید.&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:16pt;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextBrowserInteraction</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1281</width>
       <height>575</height>
      </rect>
     </property>
     <attribute name="label">
      <string>تنظیمات</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>40</y>
        <width>231</width>
        <height>131</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>20</y>
         <width>47</width>
         <height>100</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="lblStatPrepCount">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblStatCommonWordCount">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblStatVerbCount">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblStatTotalUselessWordCount">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>133</width>
         <height>100</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>تعداد حروف:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>تعداد کلمات پرکاربرد:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>تعداد افعال:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>تعداد کل کلمات بی‌اثر:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>231</width>
        <height>131</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>20</y>
         <width>47</width>
         <height>100</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="lblStatSentenceCount">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblStatWordCount">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblStatLetterCount">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_5">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>133</width>
         <height>100</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>تعداد جملات ورودی:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>تعداد کلمات:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>تعداد حروف:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>231</width>
        <height>131</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>81</x>
         <y>21</y>
         <width>103</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>تعداد کلمات موثر:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblStatEffectiveWordCount">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>21</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>40</y>
        <width>414</width>
        <height>142</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>عنوان مطلب:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="txtSettingTitle"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>آستانه‌ی تشابه جملات برای ساخت بندها:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="txtSettingSentenceThreshold"/>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>آستانه‌ی تشابه  بندها برای ساخت فصل‌ها:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="txtSettingParagraphThreshold"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnProcess">
         <property name="text">
          <string>پردازش</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1281</width>
       <height>575</height>
      </rect>
     </property>
     <attribute name="label">
      <string>آمار</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="page_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <attribute name="label">
      <string>خروجی</string>
     </attribute>
     <widget class="QTextBrowser" name="txtBrsOutput">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1251</width>
        <height>511</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOutputSave">
      <property name="geometry">
       <rect>
        <x>1100</x>
        <y>530</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>ذخیره کردن ...</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1285</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>پرونده</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>راهنما</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_2">
   <property name="text">
    <string>پروژه‌ی جدید ...</string>
   </property>
   <property name="font">
    <font>
     <family>B Nazanin</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>خروج</string>
   </property>
   <property name="font">
    <font>
     <family>B Nazanin</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>درباره‌ی ...</string>
   </property>
   <property name="font">
    <font>
     <family>B Nazanin</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="actionAddFilesToInput">
   <property name="text">
    <string>AddFilesToInput</string>
   </property>
  </action>
  <action name="actionDeleteFileFromInput">
   <property name="text">
    <string>DeleteFileFromInput</string>
   </property>
  </action>
  <action name="actionProcess">
   <property name="text">
    <string>Process</string>
   </property>
  </action>
  <action name="actionSaveOutput">
   <property name="text">
    <string>SaveOutput</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnInputAddFile</sender>
   <signal>clicked()</signal>
   <receiver>actionAddFilesToInput</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1195</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnInputDeleteFile</sender>
   <signal>clicked()</signal>
   <receiver>actionDeleteFileFromInput</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1035</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_4</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>642</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnProcess</sender>
   <signal>clicked()</signal>
   <receiver>actionProcess</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>635</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOutputSave</sender>
   <signal>clicked()</signal>
   <receiver>actionSaveOutput</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1180</x>
     <y>725</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
